Tools/WinMLRunner/dllload.cpp

```diff
@@ -0,0 +1,73 @@
+#include "FileHelper.h"
```

Comment:
This is almost the same file as [this gist](https://gist.github.com/kennykerr/3649b0878824ead2755e1f7c86e4270a), with some modifications specific to winml. This code makes it possible to use a local Windows.AI.MachineLearning.dll DLL instead of the System32 one.